### DISCUSSION TITLE
Triggerer's async thread blocked when running standalone

### DISCUSSION THREAD

**Discussion by ditirodt** [apache/airflow#28350] [Q&A]
I am using the virtualenv on ec2 and getting this error when I try to run standalone, what might be the problem, `"Triggerer's async thread was blocked for 6.38 seconds, likely by a badly-written trigger. Set PYTHONASYNCIODEBUG=1 to get more information on overrunning coroutines when running apache standalone server"`

**Comment by potiuk** [comment]
Did you follow the advice with `PYTHONASYNCIODEBUG=1` - what information did you get when you followed it ?
